klaviyo/public.py

```diff
@@ -129,8 +156,27 @@ def identify(self, email=None, external_id=None, properties={}, is_test=False):
             'properties': properties
         }
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.IDENTIFY, query_string)
+        if method.lower() not in [KlaviyoAPI.HTTP_POST, KlaviyoAPI.HTTP_GET]:
+
+            raise KlaviyoException('method argument must be either "post" or "get"')
+
+        if method == KlaviyoAPI.HTTP_POST:
```

Comment:
it looks like this if/else logic is repeated, could be a use case for a helper function so we don't need to have this written twice